### ISSUE TITLE
Handle password authentication

### ISSUE THREAD

**Issue by nfischer** [MatthieuLemoine/shelljs-plugin-ssh#1]
I'm trying this out on my ubuntu system, but the commands don't appear to be executing.

``` js
var ssh = require('shelljs-plugin-ssh');
var shell = require('shelljs');

var out = shell.ssh('nate@localhost', 'touch newFile.txt');
console.log(out);
```

I'm just trying to run a command on the same machine (I've verified that the ssh daemon is working). My machine normally requires a password to log in, but the script finishes instantly without requiring a password. The file does not get created. This is definitely recognizing my host, however, because if I supply a fake hostname, the script will hang.

**Comment by MatthieuLemoine** [comment]
Actually [ssh2-client](https://github.com/MatthieuLemoine/ssh2-client), on which depends this plugin, does not support interactive password input for now. It only relies on ssh-agent and/or a private key/password you can pass in an options object. You can file an issue on this repo too.

I will definitely implement this in ssh2-client very soon.

**Comment by MatthieuLemoine** [comment]
This is fixed in v1.2.0.
See b0e9613.

To enable interactive password input you have to pass the `promptPassword` option : 

``` javascript
var ssh = require('shelljs-plugin-ssh');
var shell = require('shelljs');

var out = shell.ssh('nate@localhost', 'touch newFile.txt', {
  promptPassword : true
});
console.log(out);
```

**Comment by nfischer** [comment]
Thanks! I'll try this out tonight.
